src/services/members.js

```diff
@@ -84,6 +85,31 @@ class MembersService {
     return getApiResponsePayloadV3(res);
   }
 
+  /**
+   * Gets member statistics history
+   * @param {String} handle
+   * @return {Promise} Resolves to the stats object.
+   */
+  async getStatsHistory(handle) {
+    const res = await this.private.api.get(`/members/${handle}/stats/history`);
+    return getApiResponsePayloadV3(res);
+  }
+
+  /**
+   * Gets member statistics distribution
+   * @param {String} handle
+   * @param {String} track
+   * @param {String} subTrack
+   * @return {Promise} Resolves to the stats object.
+   */
+  async getStatsDistribution(handle, track, subTrack) {
+    const res = await this.private.api.get(`/members/stats/distribution?filter=${encodeURIComponent(qs.stringify({
```

Comment:
No need for `encodeURICompnent` here: `qs` does it automatically, by default (https://www.npmjs.com/package/qs#stringifying)